.github/workflows/release.yml

```diff
@@ -215,5 +216,5 @@ jobs:
           fi
 
 permissions:
-  contents: write
+  contents: read
```

Comment:
This wasn't mentioned in the PR description. I know we've had to set this field to `write` in our workflows explicitly previously but that it was unexpected. Why is it safe to change it back now?